### PR TITLE
Overrides stories and tests

### PR DESCRIPTION
**Pull request by weili-govtech**
**Changes**
- final batch of commits to merge to the long-lived branch
- added stories and tests for overrides
- created overrides story template

### CODE REVIEW

**Review by qroll — CHANGES_REQUESTED** [review]

File: src/__tests__/components/fields/checkbox-group/checkbox-toggle-group.spec.tsx [hunk 1]
```diff
@@ -203,7 +183,63 @@ describe("checkbox toggle group", () => {
 		`(
 			"$scenario",
 			async ({ selected, expectedValueBeforeUpdate, expectedValueAfterUpdate }: Record<string, string[]>) => {
-				render(<CustomComponent />);
+				render(
+					<ComponentWithSetSchemaButton
+						onClick={(data) => ({
+							...data,
+							overrides: {
```
[the remark this answers is not in the document]
Comment (weili-govtech) [reply src/__tests__/components/fields/checkbox-group/checkbox-toggle-group.spec.tsx]: ok have fixed it

File: src/__tests__/components/fields/radio-button/radio-image-button.spec.tsx [hunk 2]
```diff
@@ -116,57 +138,66 @@ describe("radio toggle button", () => {
...
 			${"should clear field value if option is removed on schema update"}      | ${"B"}   | ${"Berry"}                | ${""}
 		`(
 			"$scenario",
 			async ({ selected, expectedValueBeforeUpdate, expectedValueAfterUpdate }: Record<string, string>) => {
-				render(<CustomComponent />);
+				render(
+					<ComponentWithSetSchemaButton
+						onClick={(data) => ({
+							...data,
+							overrides: {
```
[the remark this answers is not in the document]
Comment (weili-govtech) [reply src/__tests__/components/fields/radio-button/radio-image-button.spec.tsx]: have this as well

File: src/__tests__/components/fields/radio-button/radio-toggle-button.spec.tsx [hunk 3]
```diff
@@ -142,7 +142,7 @@ describe("radio toggle button", () => {
 		expect(getRadioButtonB()).toBeDisabled();
 	});
 
-	describe("update options schema", () => {
+	describe("update options through schema", () => {
```
Comment (qroll) [inline src/__tests__/components/fields/radio-button/radio-toggle-button.spec.tsx:145]: the schema merge is still using the overrides here

Comment (weili-govtech) [reply src/__tests__/components/fields/radio-button/radio-toggle-button.spec.tsx:145]: whoops just fixed that too

**Review by qroll — APPROVED** [review]
looked through the stories, they seem to be ok

**Pull request merged**